open_ce/container_build.py

```diff
@@ -219,6 +219,9 @@ def build_with_container_tool(args, arg_strings):
     Create a build image and run a build inside of container based on that image.
     """
 
+    if not args.container_tool:
```

Comment:
I was thinking if it is needed to check the existence of tool which user has provided here.